lib/close_old_pull_requests.rb

```diff
@@ -64,20 +62,20 @@ def initialize(github)
     end
 
     def clean_old_pull_requests
```

Comment:
I've added a new `Summary` class in https://github.com/everypolitician/close_old_pull_requests/pull/13/commits/7ef6da241c46d5228fb01fef5e75b25538bdb331. I think it's a step in the right direction, though possibly a bit of a big change for this pull request?